<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newPersonDialog</class>
 <widget class="QDialog" name="newPersonDialog">
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
     <property name="leftMargin">
      <number>1</number>
     </property>
     <property name="topMargin">
      <number>1</number>
     </property>
     <property name="rightMargin">
      <number>1</number>
     </property>
     <property name="bottomMargin">
      <number>1</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="jmenoLabel">
           <property name="text">
            <string>Jmeno: </string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="nameEdit"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="prijmeniLabel">
           <property name="text">
            <string>Prijmeni: </string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="surnameEdit"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="denNarozeniLabel">
           <property name="text">
            <string>Den narozeni: </string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="dayEdit"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="mesicLabel">
           <property name="text">
            <string>Mesic: </string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="monthEdit"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="rokLabel">
           <property name="text">
            <string>Rok: </string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="yearEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QCalendarWidget" name="calendarWidget"/>
         </item>
         <item>
          <widget class="QDateEdit" name="dateEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="buttonCancel">
         <property name="text">
          <string>Zavrit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonOk">
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
